docker/jupyterlab/Dockerfile.rok

```diff
@@ -0,0 +1,21 @@
+# Use tensorflow-1.14.0 as a base image, allowing the user to
+# speficy if they want GPU support, by setting IMAGE_TYPE to "gpu".
+FROM gcr.io/arrikto-public/tensorflow-1.14.0-notebook-cpu:kubecon-workshop
+
+USER root
+
+RUN echo 'alias grep="grep --color=auto' >> /etc/bash.bashrc && \
+    echo 'alias ls="ls --color=auto' >> /etc/bash.bashrc
+
+# Install latest KFP SDK & Kale & JupyterLab Extension
+RUN pip3 install --upgrade pip && \
+    pip3 install https://storage.googleapis.com/ml-pipeline/release/latest/kfp.tar.gz --upgrade && \
+    pip3 install kubeflow-kale && \
+    pip3 install jupyterlab==1.1.4 --upgrade && \
```

Comment:
I propose to use instead `pip3 install 'jupyterlab<2.0.0'. Also, `--upgrade` should not be needed when specifying version number?

They should release within a month version `2.0.0`. I think we should be good with any version `<2.0.0`. If we write it like this and they update with minor version we will have automatically the latest package.